F124663 add TLS 1.3 and associated cipher suite support

markgxchen opened the pull request:
**Description of the change**

<!-- Describe the scope of your change - i.e. what the change does. -->

**Benefits**

<!-- What benefits will be realized by the code change? -->

**Drawbacks**

<!-- Describe any known limitations with your change -->

**Applicable issues**

<!-- Enter any applicable Issues here (You can reference an issue using #) -->
  - fixes #

**Additional information**


**Checklist** <!-- [Place an '[X]' (no spaces) in all applicable fields. Please remove unrelated fields.] -->
- [] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [] Variables are documented in the README.md
- [] Title of the PR starts with chart name (e.g. `[charts/gateway]`)
- [] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

markgxchen committed:
F124663 add TLS 1.3 and associated cipher suite support

cuisimon commented on charts/portal/templates/pssg/pssg-config.yaml:
```diff
@@ -38,3 +38,8 @@ data:
   PORTAL_VERSION: {{ .Chart.AppVersion }}
   SSG_JVM_HEAP: 4096m
+{{ if .Values.pssg.additionalEnv }}
+{{- range $key, $val := .Values.dispatcher.additionalEnv }}
```
.Values.**dispatcher**.additionalEnv should be pssg

markgxchen replied:
Good catch. Thanks

markgxchen committed:
F124663 add TLS 1.3 and associated cipher suite support

markgxchen committed:
F124663 iTLS 1.3 support
 add README for TLS support additionalEnv
 add additionalEnv to values-production.yaml

markgxchen committed:
F124663 TLS 1.3 support - update README

melil02 reviewed (approved):
LGTM
